### DISCUSSION TITLE
How to tweak maxtheme file?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [MagistraNocte/Max-8-Default-Dark-Mode#1] [Q&A] [answered]
I want to customize some colors for my taste.
Is it possible? then How to do it?
Thanks !

**Comment by MagistraNocte** [comment, marked as answer]
Hi, sorry for the big delay, I don't check github often.
You can open the maxtheme file with any text editor (I used Textwrangler on mac) inside you can see a list of entries separated by curly brackets, every entry has an ID, a category and possibly a label (these should give you an idea of what that color is used for, but you're gonna have to do a lot of trial and error) and a color code in RGBA format, you just change that code and save. At the end of the file (in the styledefaults section) there is a list of the most common colors used in the max ui, don't forget to experiment with those too. after saving select the theme again in the max preferences to make changes effective.
